### ISSUE TITLE
Re-trying report of Fen issue

### ISSUE THREAD

**Issue by UPRHome** [Tikipeter/repository.tikipeter#322]
Sorry - Forgot to add log file last time:
Noticing that similar to a previously posted issue that pre-selected items from Fen (in this case from Favourites screen/tab but previously as widgets) that the selections will not work. A favourite TV show saved from Fen in the Favourites section will go through the seasons and episodes, but when clicking on the episode, no results available. If instead one manually searches from the addon (Fen / TV Shows / Trending (eg) / Specific TV show (i.e. NCIS / Season x / Episode X then the Fen addon searches for available streams normally and will play. Also, in many cases, when accessing a specific TV show saved from Fen in Favourites, it will not show newly available seasons. Have tried clearing various caches etc to no avail.

Was not able to reproduce links from Favourites not playing but did find that accessing TV show (NCIS) would not show all seasons (most recent season 18) - see link below

https://www.dropbox.com/s/ed7umsw2tkfpk37/kodi.log?dl=0
(Hope I'm doing this correctly)

**Comment by Tikipeter** [comment]
Please turn off Reuse Language Invoker and see if that fixes the issue (Fen->Tools->Set Language Invoker (ADVANCED!!)). It's at the bottom of Tools. Set it to OFF and try again.

Also, report please if you are using Kodi 18.9 (your log was cut at the start and doesn't have that info). 18.9 seems to be having all kinds of issues.

**Comment by UPRHome** [comment]
Yes, that's right I'm not using Fen Favourites but had added the show to the Kodi favourites some time ago (at least a month or two). I can also get shows to play, but it's odd with the particular TV show (NCIS) I was using as an example that even with that change (Language Invoker) you suggested, and again clearing all caches, even from the main search through the addon (not from the Favourites) when I go to NCIS, I still can't get it to show Season 18 (found that season through another addon). Weird.

**Comment by Tikipeter** [comment]
I don't know what Kodi version ships with LibreELEC 9.2.6. Doesn't matter though.

I can't get straight what the issue is here, because now you're saying you can get episodes to play, but one particular series (NCIS) doesn't show the most recent season. which does show for me (Season 18).

I'm going to need specific examples of specific issues, because at the moment you've given two different issues in the two posts you have made. Remember I am not looking at your setup when you write down the problem.

i.e.
* I made "some show" a favorite and it won't let me browse into it.
* I made "some show" a favorite and it won't display results for episodes.
* "some show" doesn't display the latest season. Nothing to do with Favorites though.

etc.

At the moment it's all over the place, and I don't know where to start.

**Comment by UPRHome** [comment]
My apologies for the confusion.  Just trying to figure out specifically what it was or wasn't doing was a bit tricky. (my bad)
I'm attaching a new log. (see below)
The problem seems to have something to do with existing TV shows saved to Kodi favourites from Fen.

Here is what I did:

Called up a TV show from Kodi favourites called "For Life"
There are two seasons (they are just starting 2nd season) - The second season was not showing.
I removed my favourites.xml file and rebooted unit.
I uninstalled Fen and relevant files from device and rebooted.
Reinstalled Fen.
Re-synced Fen to My Accounts (am using Real-Debrid)
Searched for the TV show "For Life" using standard search and now both seasons showed in search.
Reloaded my favourites.xml file
Rebooted the device.
Checked the Kodi favourites screen (now showing all of my original favourites again) and selected "For Life" TV show.
Second season was now showing for that show.
Selected an episode and the search started, said "Preparing sources" and then "No Results"
Came out of Favourites and searched for the same show through search function.
Tried an episode and it did a normal search, found results and it played normally.
Since then, I have noticed that all other favourites previously saved to Kodi favourites from Fen do not work.

https://www.dropbox.com/s/ed7umsw2tkfpk37/kodi.log?dl=0
